<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContactWindow</class>
 <widget class="QDialog" name="AddContactWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>216</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="LabelFirstName">
     <property name="styleSheet">
      <string notr="true">QWidget{
	font: 8pt &quot;IRANSans&quot;;
}</string>
     </property>
     <property name="text">
      <string>نام : </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="LabelLastName">
     <property name="styleSheet">
      <string notr="true">QWidget{
	font: 8pt &quot;IRANSans&quot;;
}</string>
     </property>
     <property name="text">
      <string>نام خانوادگی : </string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="TxtFirstName">
     <property name="styleSheet">
      <string notr="true">QLineEdit{
margin:10px;
padding:5px;
	background-color: rgb(240, 240, 240);
	border:1px solid;
	border-color: rgb(200, 200, 200);
	border-radius:5px;
	font: 8pt &quot;IRANSans&quot;;
}</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="LabelPhone">
     <property name="styleSheet">
      <string notr="true">QWidget{
	font: 8pt &quot;IRANSans&quot;;
}</string>
     </property>
     <property name="text">
      <string>تلفن : </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="TxtLastName">
     <property name="styleSheet">
      <string notr="true">QLineEdit{
margin:10px;
padding:5px;
	background-color: rgb(240, 240, 240);
	border:1px solid;
	border-color: rgb(200, 200, 200);
	border-radius:5px;
	font: 8pt &quot;IRANSans&quot;;
}</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="TxtPhone">
     <property name="styleSheet">
      <string notr="true">QLineEdit{
margin:10px;
padding:5px;
	background-color: rgb(240, 240, 240);
	border:1px solid;
	border-color: rgb(200, 200, 200);
	border-radius:5px;
	font: 8pt &quot;IRANSans&quot;;
}</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="BtnSubmit">
     <property name="styleSheet">
      <string notr="true">QPushButton:pressed {
     background-color:   rgb(102, 102, 102);
	color: rgb(240, 240, 240);
}
QPushButton {
     background-color:  rgb(240, 240, 240);
	color: rgb(102, 102, 102);
border: 1px solid  rgb(102, 102, 102);
     border-radius: 5px;
padding:6px;
margin:0px;
margin-left:10px;
}

QPushButton:disabled {
	color: rgb(180, 180, 180);
}</string>
     </property>
     <property name="text">
      <string>افزودن</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
